spanner/tests/system/test_system.py

```diff
@@ -219,20 +219,22 @@ def _assert_timestamp(self, value, nano_value):
         else:
             self.assertEqual(value.microsecond * 1000, nano_value.nanosecond)
 
-    def _check_row_data(self, row_data, expected=None):
+    def _check_table_data(self, table_data, expected=None):
```

Comment:
This name is wrong: we aren't always checking a whole table. if you must split out the checks for a single row (why?), then call this `_check_rows_data`.